src/devices/Board/Board.cs

```diff
@@ -0,0 +1,407 @@
+﻿using System;
```

Comment:
you're missing copyright header in couple of files

Reply:
Yes, and documentation as well, I suppose